hybridauth.routing.yml

```diff
@@ -23,3 +23,11 @@ hybridauth.settings:
     _title: 'HybridAuth'
   requirements:
     _permission: 'administer site configuration'
+
+hybridauth.providers.settings:
```

Comment:
I think it should be `hybridauth.provider.settings`

Reply:
Changed. Thanks.